thirdparty/patches/rocksdb-6.22.1-metadata-header.patch

```diff
@@ -0,0 +1,84 @@
+diff --git a/include/rocksdb/metadata.h b/include/rocksdb/metadata.h
```

Comment:
refer to https://github.com/facebook/rocksdb/commit/c9cd5d25a832524f09af3276d29f0d9499eb58ad for the relative change.